### PR TITLE
chore(cdk): `migrations` remove deprecated interfaces

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [x] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes # 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/cdk/schematics/ng-update/tests/schematic-rename-types.spec.ts [hunk 1]
```diff
@@ -35,6 +35,7 @@ describe('ng-update', () => {
 import { TuiButtonOptions, some } from '@taiga-ui/core';
 
 const options: TuiButtonOptions = {};
+const config  = {};
```
Comment (nsbarsukov) [inline projects/cdk/schematics/ng-update/tests/schematic-rename-types.spec.ts:38]:
```suggestion
const config = {};
```

Comment (nsbarsukov) [reply projects/cdk/schematics/ng-update/tests/schematic-rename-types.spec.ts:38]: I think
```ts
const config: WithDateMaskPipeConfig = {}
```
should become
```ts
const config: any = {};
```

Why:
Imagine user has
```ts
const config: TuiBreadCrumbsItem = {caption: '', routerLink: ''};
// ...

console.log(config.routerLinkOptions);
```
after migration:
```ts
const config = {caption: '', routerLink: ''};
// ...

// TS2339: Property 'routerLinkOptions' does not exist on type ...
console.log(config.routerLinkOptions);
```

**Review by nsbarsukov — CHANGES_REQUESTED** [review]

File: projects/cdk/schematics/ng-update/tests/schematic-rename-types.spec.ts (hunk 1 above)
Comment (vladimirpotekhin) [reply projects/cdk/schematics/ng-update/tests/schematic-rename-types.spec.ts]: Nice idea, thanks:)

**Review by nsbarsukov — APPROVED** [review]

**Review by splincode — APPROVED** [review]

File: projects/cdk/schematics/ng-update/steps/rename-types.ts [hunk 2]
```diff
@@ -1,27 +1,44 @@
...
-            const namedImport = parent
-                .getImportDeclaration()
-                .getNamedImports()
-                .find(specifier => specifier.getName() === from);
-            namedImport?.replaceWithText(to);
-        } else if (Node.isTypeReferenceNode(parent)) {
-            parent.replaceWithText(to);
+        if (Node.isImportSpecifier(parent) && to) {
+            renameImport(parent, to, from);
+            return;
```
Comment (waterplea) [inline projects/cdk/schematics/ng-update/steps/rename-types.ts:24]: Maybe chain it with `else if` so there are no returns?

**Review by waterplea — APPROVED** [review]

**Pull request merged**